src/test/ValidationFeiRiskParamsUpdate.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+import "forge-std/Test.sol";
+import "forge-std/console.sol";
+import {AaveV2Helpers, ReserveConfig} from "./utils/AaveV2Helpers.sol";
+import {AaveGovHelpers, IAaveGov} from "./utils/AaveGovHelpers.sol";
+
+import {FeiRiskParamsUpdate} from "../FeiRiskParamsUpdate.sol";
+
+contract ValidationFeiRiskParamUpdate is Test {
+    address internal constant AAVE_WHALE =
+        0x25F2226B597E8F9514B3F68F00f494cF4f286491;
+
+    address internal constant FEI = 0x956F47F50A910163D8BF957Cf5846D573E7f87CA;
+
+    // can't be constant for some reason
+    string internal MARKET_NAME = "AaveV2Ethereum";
+
+    function setUp() public {}
+
+    /// @dev Uses an already deployed payload on the target network
+    function testProposalPostPayload() public {
+        /// deploy payload
+        FeiRiskParamsUpdate fei = new FeiRiskParamsUpdate();
+        address payload = address(fei);
+        _testProposal(payload);
```

Comment:
Small nitpick. This line seems unnecessary. You could make it just `address(fei)` in the `_testProposal()` argument.